BreadcrumbTrail/Trail.php

```diff
@@ -203,7 +203,8 @@ public function add($breadcrumb_or_title, $routeName = null, $routeParameters =
 
             $url = null;
             if ( !is_null($routeName) ) {
-                $url = $this->router->generate($routeName, $routeParameters, $routeAbsolute);
+                $referenceType = $routeAbsolute ? UrlGeneratorInterface::ABSOLUTE_PATH : UrlGeneratorInterface::RELATIVE_PATH;
```

Comment:
You mean 
` $routeAbsolute ? UrlGeneratorInterface::ABSOLUTE_URL : UrlGeneratorInterface::RELATIVE_PATH;`
instead, don't you ?